README.md

```diff
@@ -29,3 +35,13 @@ The actual contents depends on the project you have.
 
 If you have 3rd party dependencies, their licenses should be placed in
 `/licenses-dependencies/<name-of-dependency>`
+
+### Set up dependabot
+
+Create a new label Github label `dependencies` by following
```

Comment:
```suggestion
Create a new Github label `dependencies` by following
```

Reply:
There can never be enough labels labels labels!

Will fix